packages/wallets/src/components/Loader/Loader.scss

```diff
@@ -1,63 +1,71 @@
 .wallets-loader {
     position: absolute;
     display: flex;
-    aspect-ratio: 2/1;
+    width: 7.2rem;
+    height: 3.2rem;
     transition: all 0.4s ease;
     flex-direction: row;
     justify-content: space-between;
     align-items: center;
-    height: 2rem;
     top: 50%;
     left: 50%;
     transform: translate(-50%, -50%);
 
     &__element {
-        $loader: loader 1.2s ease infinite both;
+        $loader: loader 1.5s ease-in infinite both;
         display: block;
         align-items: flex-end;
-        width: 0.5rem;
-        height: 25%;
+        width: 0.8rem;
+        height: 3.2rem;
         animation: $loader;
-        border-radius: 0.5rem;
+        border-radius: 1rem;
         bottom: 0;
 
         &:nth-child(2) {
-            animation-delay: 0.4s;
+            animation-delay: 0.1s;
         }
 
         &:nth-child(3) {
-            animation-delay: 0.8s;
+            animation-delay: 0.2s;
+        }
+
+        &:nth-child(4) {
+            animation-delay: 0.3s;
+        }
+
+        &:nth-child(5) {
+            animation-delay: 0.4s;
         }
```

Comment:
&:nth-child(n) {
    animation-delay: calc((n - 1) * 0.1s);
}

Reply:
It seems like n does not get passed and the calc does not take place. Any idea on how to improve on that?

https://github.com/binary-com/deriv-app/assets/142860499/ce63e03c-954b-4c9a-8b90-bc4c50f25d16

Reply:
Resolved 👍